src/main/resources/default/kb/de/admin/core.html.pasta

```diff
@@ -0,0 +1,2 @@
+<k:chapter code="SKAM" lang="de" priority="100" title="Kernfunktionen" permissions="" parent="JFLRO">
```

Comment:
code has only 4 chars - all others have 5.
Wouldn´t it be better if they have speaking codes like "KERNF" or "ADMIN"

Reply:
Nope, we plant for drastically grow this and do not want to spend hours in imagining short names - therefore these are _by design_ pointless codes..